components/server/src/code-sync/code-sync-service.ts

```diff
@@ -114,12 +114,7 @@ export class CodeSyncService {
                 return;
             }
             const manifest = await this.db.getManifest(req.user.id);
-            if (!manifest.latest.extensions) {
```

Comment:
why is it removed?

Reply:
Thanks for noting this, @akosyakov, I haven't really thought about the fact it could still be used. I will re-add it and we can perhaps discuss it another time, it is indeed out-of-scope for this PR.

Reply:
> If you have a suer who used Theia and has user settines/extensions which are portable to VS Code and he start with VS Code now then we lose all his data?

@akosyakov  Is it possible to add a check so that we are sure user don't have theia data, so we can avoid returning new entries with `fromTheiaRev`

Reply:
I think we should move this discussion to own issue and discuss it there.

It is not possible to add check here, you will need to run query on database to see how many users with Theia data which were not overridden by VS Code data.

Reply:
Reverted the changes ✅